src/util/i18nFormatting.js

```diff
@@ -0,0 +1,24 @@
+const formatFlags = {
+  EN_HANDLE_AN: 'en-handle-an',
+  CAPITALIZE: 'capitalize'
+}
+
+function getVariationOfAOrAn(value) {
+  const vowelishLetters = ['a', 'e', 'i', 'o', 'u', 'h'];
```

Comment:
We can probably move this into the top level of the module scope to avoid reconstructing it each time the function is called?

Also, maybe use a `Set` so that we can do O(1) lookup? (Obviously this is probably neither here nor there for a six-element collection)